integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+// Mount a vfat volume and kexec the kernel within.
+func main() {
+	// Test : Mount on non-existent directoty
+	if err := mountkexec.MountISO("/dev/sda1", ""); err != nil {
+		log.Printf(err.Error())
+	}
+	// Test : Mount non-existent iso
+	if err := mountkexec.MountISO("non-existent file", "mountDir"); err != nil {
+		log.Printf(err.Error())
+	}
+	// Test : Mount on existing dir and an existing iso
+	if err := mountkexec.MountISO("/dev/sda1", "mountDir"); err != nil {
+		log.Fatal(err)
+	}
+	//Test : Check files in mounted iso
+	o, err := exec.Command("ls", "mountDir/").CombinedOutput()
+	if err != nil {
+		log.Fatalf("ls failed: error %v", err)
+	} else {
```

Comment:
you don't need the else. log.Fatalf will exit.

Reply:
I am using the else to make sure that the out put from running ls is returned to mountkexec_test.go.The results from running the command ls are stored in the object which I am printing in that else statement .

Reply:
The log print is fine. But since the Fatalf exits, you can remove 
} else {
and just have the log.Printf()

Reply:
did you this note? you can replace
} else {
		log.Printf("%v", string(o))
	}
with
		log.Printf("%v", string(o))

Reply:
Done.